bindings/wasm/package.json

```diff
@@ -72,7 +72,9 @@
     "wasm-opt": "^1.3.0",
     "wasm-pack": "0.10.1",
     "webpack": "^5.67.0",
-    "webpack-cli": "^4.9.2"
+    "webpack-cli": "^4.9.2",
+    "big-integer": "^1.6.51",
+    "@iota/util.js": "^1.9.0-stardust.5"
```

Comment:
Nit: try keep dependencies alphabetically ordered and move `@iota/util.js` below `@iota/crypto.js`.